### PR TITLE
Fix: boxes in let bindings

### PR DESCRIPTION
**Pull request by gpetiot**
From JS fork (https://github.com/janestreet/ocamlformat/commit/684bd4f54e1c5a69b0a933417c359aeb7140501f)

### CODE REVIEW

File: test/passing/fun_decl.ml [hunk 1]
```diff
@@ -12,47 +12,47 @@ class t =
...
-    aaaaaaaaaaa
-    bbbbbbbbbbbbbb
-    ccccccccccccccccccc
-    ddddddddddddddddddddd
-    eeeeeeeeeeeeeee =
+       aaaaaaaaaaa
+       bbbbbbbbbbbbbb
+       ccccccccccccccccccc
+       ddddddddddddddddddddd
+       eeeeeeeeeeeeeee =
```
Comment (jberdine) [inline test/passing/fun_decl.ml]: Is this really intended?

**Comment by gpetiot** [comment]
Is this good to be merged?

File: test/passing/fun_decl.ml [hunk 2]
```diff
@@ -12,47 +12,47 @@ class t =
...
-    aaaaaaaaaaa
-    bbbbbbbbbbbbbb
-    ccccccccccccccccccc
-    ddddddddddddddddddddd
-    eeeeeeeeeeeeeee =
+   aaaaaaaaaaa
+   bbbbbbbbbbbbbb
+   ccccccccccccccccccc
+   ddddddddddddddddddddd
+   eeeeeeeeeeeeeee =
```
Comment (jberdine) [inline test/passing/fun_decl.ml]: I don't understand these changes, but am assuming that they are intended. Is there some discussion that motivates indenting the arguments an odd number of spaces, i.e. in between two natural "tab stops", and furthermore indents the arguments less than the function name?

These changes are nicely guarded, so I don't object to merging this, but just for the record I don't get it. :-)

Comment (jberdine) [reply test/passing/fun_decl.ml]: So @hhugo, approve this PR is it's really what you want.

Comment (gpetiot) [reply test/passing/fun_decl.ml]: I don't get the indentation either but it comes directly from the JaneStreet fork so I assume they have a use for it. But maybe the indentation could be parameterized with another option like `indent-fun-args=N` ? (with N=3 for JaneStreet)

**Comment by gpetiot** [comment]
Is this one good to go or do you think we need an option to parameterize the indent?

**Comment by jberdine** [comment]
Also for this one, I think @hhugo is the one who needs to make the call on this one.

**Comment by gpetiot** [comment]
This style does not seem to match JS style afterall, maybe just close this PR?

**Pull request closed without merging**